packages/renderer/src/lib/image/ImageDetailsFiles.svelte

```diff
@@ -18,36 +18,56 @@ let selectedLayer = $state<ImageFilesystemLayerUI>();
 let loading = $state<boolean>(false);
 let error = $state<string>('');
 let showLayerOnly = $state<boolean>(false);
+let showFetchButton = $state<boolean>(false);
 
 let filesProvidersUnsubscribe: Unsubscriber;
-let filesProvider: ImageFilesInfo;
+let filesProvider: ImageFilesInfo | undefined = undefined;
 let cancellableTokenId: number = 0;
+let askFetchLayers: boolean = true;
 
-function onSelectedLayer(event: CustomEvent<ImageFilesystemLayerUI>) {
+function onSelectedLayer(event: CustomEvent<ImageFilesystemLayerUI>): void {
   selectedLayer = event.detail;
 }
 
+async function fetchImageLayers(provider: ImageFilesInfo, img: ImageInfo): Promise<void> {
+  try {
+    loading = true;
+    cancellableTokenId = await window.getCancellableTokenSource();
+    const layers = await window.imageGetFilesystemLayers(provider.id, img, cancellableTokenId);
+    imageLayers = layers;
```

Comment:
just wondering if we need to use this intermediate variable

Reply:
No, removing it